static/css/components/cbox.less

```diff
@@ -12,6 +12,13 @@
   z-index: @z-index-level-17;
 }
 
+#cboxContent {
+  width: auto !important;
+}
+#cboxLoadedContent{
+  height: auto !important;
```

Comment:
Use of !important is an anti-pattern we should avoid. We should use it only as a last resort and if we are using it for good reason we should document it in great detail (off topic: we should document this in Contributing.md)

In this particular case you are trying to override heights and widths defined by a JS library and I don't recommend doing this. You are fighting against some code we don't understand fully.

Instead, try to style .floatform to get the desired effect!

Reply:
@jdlrobson I understood the real problem!
Inside colorbox, .floater elements have their widths hardcoded in cbox.less 
`div.floater {
  position: relative;
  font-family: @lucida_sans_serif-1;
  width: 640px;
  min-height: 670px;
........ }
`
But the default value of border-sizing of colorbox is `box-sizing: border-box`. This is causing the problem. As we are giving `width:640px` in .floater class we need `box-sizing` rule of colorbox to be content-box.
[Ref]
So just one addition `box-sizing:content-box` in colorbox will make things work without modifying CSS of it's child elements.

Would this be fine?
[Ref] https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing

Reply:
@jdlrobson I have updated the PR.

Changes are:
-Removed unnecessary `top-margin: 31` in `div.tabsPop` 
-Added margin on `div.imageIntro`
-Added `width: auto !important; height: auto !important` in `#cboxContent`. 

About !important usage,
I understand that it's usage should be avoided but colorbox generates `div#cboxContent` with inline CSS rules for width and height. So adding !important was the only way out. If you find any alternative, I will update the PR again.

[inline css in #cboxContent]
![WhatsApp Image 2019-03-12 at 6 00 01 PM](https://user-images.githubusercontent.com/32803230/54200163-b6ba1900-44f0-11e9-8277-85338d8356b7.jpeg)